examples/hybrid/cli_options.jl

```diff
@@ -207,6 +207,10 @@ function parse_commandline()
         help = "Define the surface elevation profile [`NoWarp`,`Earth`,`DCMIP200`]"
         arg_type = String
         default = "NoWarp"
+        "--apply_limiter"
+        help = "Apply a horizontal limiter to every tracer [`true` (default), `false`]"
+        arg_type = Bool
+        default = true
```

Comment:
Due to `@warn "apply_limiter flag is ignored for OrdinaryDiffEq algorithms"`, I might suggest making the default `false` and making it an opt-in option for now. As the ARS scheme improves and we replace Rosenbrock23 with ARS, then we can try making this the default. This way we can experiment with it and know exactly when it's getting used. Also, this means that we can change that `@warn` to `error("...")` so that people don't get confused about whether its being used or not

Reply:
Yeah, I thought about it too. What do you think, @dennisYatunin ?

Reply:
It's enable only when ARS is used (and that is an opt-in now). So effectively it is an opt-in, but probably not super transparent this way

Reply:
Well I was told that we want the default to be ARS343 with limiters and FCT. Since we temporarily made ODE.Rosenbrock23 the default, we could also make no limiters the default, while keeping in mind that we'll change it as soon as performance improves.